pkg/app/piped/executor/cloudrun/cloudrun.go

```diff
@@ -148,12 +148,21 @@ func revisionExists(ctx context.Context, client provider.Client, revisionName st
 	return false, err
 }
 
-func addBuiltinLabels(sm provider.ServiceManifest, hash, pipedID, appID string) {
+func addBuiltinLabels(sm provider.ServiceManifest, lp executor.LogPersister, hash, pipedID, appID, revisionName string) bool {
 	labels := map[string]string{
 		provider.LabelManagedBy:   provider.ManagedByPiped,
 		provider.LabelPiped:       pipedID,
 		provider.LabelApplication: appID,
 		provider.LabelCommitHash:  hash,
 	}
+	// Set builtinLabels for Service.
 	sm.AddLabels(labels)
+
+	// Set buildinLabels for Revision.
+	labels[provider.LabelRevisionName] = revisionName
+	if err := sm.AddRevisionLabels(labels); err != nil {
```

Comment:
I'm a bit worried here, looks like the labels pointer changed after this L162 (add one more key) and we do that on purpose in order to make sure `RevisionLabels` and `Labels` are different, right?
But with the same labels object, change on L162 will affect the labels we added to sm.Labels (ref: https://go.dev/play/p/dy5xoUlFINK). Sorry if I missed something, but I guess better to add test for this function to ensure we get what we want, it's a standalone function, so looks like adding test for it is not too annoying any way 😄

Reply:
@khanhtc1202 
Thank you for your reviewing!
Sorry, I couldn't got you perfectly, but I added test.
With that, wdyt?